source/jsonnet/northern-ireland/individual/blocks/employment/place_of_work_elsewhere.jsonnet

```diff
@@ -54,6 +54,58 @@ local pastProxyTitle = {
     },
   ],
   routing_rules: [
+    {
+      goto: {
+        block: 'town-and-county',
+        when: [
+          {
+            id: 'place-of-work-elsewhere-answer',
+            condition: 'equals any',
+            values: [
+              'Carlow',
```

Comment:
More a question, is there anything special about this list? Just wondering if it would ever be reused

Reply:
I will check with @donnap573

Reply:
I checked with Donna, there is no requirement to use this list anywhere else